docs/section11/Rule11-7.md

```diff
@@ -37,6 +37,8 @@
   - use get_SWH_components_associated_with_each_SWH_bat to get the SWH use types and SWH use in the building: `swh_bats_and_uses_b = get_SWH_components_associated_with_each_SWH_bat(B_RMD)`
   - look at each SWH Equipment in the swh_bat.  Appendix G requires only one system per bat, but this is covered in rule 11-8, so here we will allow multiple SWH equipment: `for swh_equip_id in swh_bats_and_uses_b[swh_bat]["SWHHeatingEq"]:`
     - get the SWH equipment type using the function get_SWH_equipment_type: `swh_equip_type = get_SWH_equipment_type(B_RMD, swh_equip_id)`
+    - if the swh_equip type is PROPANE_INSTANTANEOUS, change it to GAS_INSTANTANEOUS for the purposes of this rule (whether Propane is an appropriate fuel source is covered in rule 11-16): `if swh_equip_type == "PROPANE_INSTANTANEOUS": swh_equip_type = "GAS_INSTANTANEOUS"
+    - if the swh_equip type is PROPANE_STORAGE, change it to GAS_STORAGE for the purposes of this rule (whether Propane is an appropriate fuel source is covered in rule 11-16): `if swh_equip_type == "PROPANE_STORAGE": swh_equip_type = "GAS_STORAGE"
```

Comment:
If we see the [ashrae_90_1_table_G3_1_1_2.json,](https://github.com/pnnl/ruleset-checking-tool/blob/develop/rct229/rulesets/ashrae9012019/data/ashrae_90_1_table_G3_1_1_2.json) all the ```baseline_heating_method``` values end with ```_WATER_HEATER```. Should the ```swh_equip_type``` be ```GAS_STORAGE_WATER_HEATER```?

Reply:
I think that should be fine. Recommend updating this RDS.